lib/multisigHelpers.ts

```diff
@@ -51,33 +54,32 @@ const createMultisigFromCompressedSecp256k1Pubkeys = async (
 const getMultisigAccount = async (
   address: string,
   client: StargateClient,
-): Promise<Account | null> => {
+): Promise<AccountWithPubkey | null> => {
   // we need the multisig pubkeys to create transactions, if the multisig
   // is new, and has never submitted a transaction its pubkeys will not be
   // available from a node. If the multisig was created with this instance
   // of this tool its pubkey will be available in the fauna datastore
-  let accountOnChain: Mutable<Account> | null = await client.getAccount(address);
+  const accountOnChain = await client.getAccount(address);
   const chainId = await client.getChainId();
+  if (!accountOnChain) return null;
```

Comment:
It looks like this will return null from this function if no Account is returned from the stargate client, which is a problem. There are scenarios where a multisig account will be looked up that only exists in the app's database, and we should support that.

Reply:
If an account does not exist on chain, it does not have an account number. Without account number, we cannot sign at all. So the minimum requirement to start composing a transaction is that someone sent some tokens to the multisig address, such that it gets an account number assigned. UX issues around missing account numbers are tracked in https://github.com/cosmos/cosmos-multisig-ui/issues/12.

But you are right, having an account on chain does not mean the public key is available on chain. The public key only becomes available once the account signed a transaction. This is why the code below assigns a public key either from chain or the multisig database.

The code we had before would error in case the account is missing:

```
> var accountOnChain
undefined
> accountOnChain = null;
null
> accountOnChain.pubkey = "any value here";
Uncaught TypeError: Cannot set properties of null (setting 'pubkey')
```

So my thinking was to cleanly deal with the case by returning `null` as this was a valid return value already.